src/Navigation/Header/styled.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+import { ReactComponent as LogoWebsite } from "./logo.svg"
+
+export const StyledHeader = styled.header`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 12px;
+    padding-top: 5px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px){
+        gap: 6px;
+    };
+    
+    @media(max-width: ${({ theme }) => theme.breakpoints.mobile}px){    
+        justify-content: space-evenly;
+        gap: 8px;
+    };
+`;
+
+export const Logo = styled(LogoWebsite)`
+    width: 40px;
+    height: 40px;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.smallPhone}px){
+        width: 17px;
+        height: 17px;;
+    };
+
+`;
+
+export const Title = styled.h1`
```

Comment:
Between 600 and 767px Title is moving with widow resize. I think it should stay fixed relative to the Logo?